rope/contrib/generate.py

```diff
@@ -25,7 +55,7 @@ def create_generate(kind, project, resource, offset, goal_resource=None):
         "package": GeneratePackage,
         "variable": GenerateVariable,
     }
-    generate = d.get(kind)
+    generate = d.get(kind, _Generate)
```

Comment:
This is an incorrect change; `_Generate` is supposed to be an abstract class, it isn't meant to be instantiated.

Should do this instead, to forbid invalid `kind`:

```suggestion
    generate = d[kind]
```